test/modules/TestResultCoordinator/Settings.cs

```diff
@@ -131,51 +134,34 @@ public override string ToString()
             return $"Settings:{Environment.NewLine}{string.Join(Environment.NewLine, fields.Select(f => $"{f.Key}={f.Value}"))}";
         }
 
-        internal List<ITestReportMetadata> GetReportMetadataList()
+        internal async Task<List<ITestReportMetadata>> GetReportMetadataListAsync(ILogger logger)
```

Comment:
It looks like this method should be in the TestUtil. Same for GetResultSources

Reply:
This method used a class field to persist the result from GetTwin and ReportMetadata is part of settings.  Therefore I think it is better to be in the class compared to TestUtil static class.